pkg/tag_service/tag_service.go

```diff
@@ -241,6 +242,22 @@ func (s *tagServiceServer) ResolveTag(c context.Context, tag *tagservicepb.Tag)
 	return &tagservicepb.TagMappingList{Mappings: resolvedTags}, nil
 }
 
+// Resolve a list of tags into all base-level IPs
+func (s *tagServiceServer) ListTags(c context.Context, e *emptypb.Empty) (*tagservicepb.TagMappingList, error) {
+	var resolvedTagList []*tagservicepb.TagMapping
+	tags := s.client.Keys(c, "*").Val()
+	for _, tag := range tags {
+		tagMapping, err := s.GetTag(c, &tagservicepb.Tag{TagName: tag})
+		if err != nil {
```

Comment:
I'm fine with this for now, but should we give a warning if there's an error? Or, maybe we should error out of all requests give an error (since that may mean the tag service is down or something)

Reply:
I have added a log, since returning error is not ideal for IBM plugin, where KV store use the same DB, and it would fail with logs like this : .
`GetTag default:ibm:r014-af7745bc-38d7-4f9e-9957-07c022ddcf7a: GetTag default:ibm:r014-af7745bc-38d7-4f9e-9957-07c022ddcf7a: WRONGTYPE Operation against a key holding the wrong kind of value"}null`

Hence, I have just added a log, and continuing to resolve other tags.